contracts/src/libraries/Predeploys.sol

```diff
@@ -14,6 +14,7 @@ library Predeploys {
     address internal constant Staking = 0xCCcCcC0000000000000000000000000000000001;
     address internal constant UBIPool = 0xCccCCC0000000000000000000000000000000002;
     address internal constant Upgrades = 0xccCCcc0000000000000000000000000000000003;
+    address internal constant Create3 = 0xCcCcCC0000000000000000000000000000000004;
```

Comment:
The address of Contract3 on other network is `0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf`. 
Why don't we predeploy this contract under the same address?

Reply:
Good one!